### ISSUE TITLE
Incorrect throughput reporting

### ISSUE THREAD

**Issue by gernotheiser** [au-ts/ipbench#16]
ipbench reports achieved throughput (and probably applied load) in terms of payload bits. This is misleading, as it doesn't represent what's happening on the network and makes it appear as if the system cannot keep up with load.
It should measure actual bits on the wire, i.e. including Ethernet headers.

**Comment by wom-bat** [comment]
I disagree.  Requested and Applied load and throughput are what's obtainable after protocol overhead.  ipbench is layer-1 and layer-2 agnostic.  If you enable jumbo frames outside ipbench, for instance, the overheads will change, because larger packets no longer need fragmentation.  If you run over fibrechannel, interpacket gap drops from 96 to 40 bit times ... all this is outside ipbench; all ipbench can measure is the layer 5 throughput.
